### PR TITLE
Dev Status (Git)

### PR DESCRIPTION
**Pull request by khrome**
This searches through a dir for any contained .git repos and displays them as a list, whether they are clean or not and allow you to click through. I use this to know, at a glance, what needs to be flushed when I'm in the midst of updating many modules (and potentially multiple contexts).

menu

### CODE REVIEW

File: Dev/Git/DevStatus.10m.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# <xbar.title>Dev Status</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Abbey Hawk Sparrow</xbar.author>
+# <xbar.author.github>khrome</xbar.author.github>
+# <xbar.desc>Recursive status of any discovered .git repos in the provided directory. </xbar.desc>
+# <xbar.var>string(DIRECTORY="/Users/khrome/Development"): Directory to use.</xbar.var>
+# <xbar.image></xbar.image>
```
Comment (matryer) [inline Dev/Git/DevStatus.10m.sh:9]: Please provide the URL to an image for this plugin.

Comment (matryer) [reply Dev/Git/DevStatus.10m.sh:9]: p.s. you can use the one you posted in GitHub, they stick around.

**Review by matryer — CHANGES_REQUESTED** [review]
Nice. Just needs an image.

**Comment by khrome** [comment]
Is this dead/should I close it?

**Pull request merged**